packages/clay-navigation-bar/src/__tests__/index.tsx

```diff
@@ -143,10 +145,16 @@ describe('ClayNavigationBar', () => {
 		);
 
 		fireEvent.click(getByTestId('navbarToggler'), 'Trigger Label');
-		fireEvent.transitionEnd(getByTestId('NavigationBarDropdown'));
+
+		act(() => {
+			jest.runAllTimers();
+		});
```

Comment:
How come you have to wrap this in an act?

Reply:
This was throwing some errors on the console, as we are using `setTimeout` and has scheduled a function that will change the states, this probably causes new renderings later and we have to wrap this within an `act`.

Reply:
Hmm, that seems odd. I would think you could just run it without the `act`.

Oh well, im fine with merging!

Reply:
Yeah, I remember that I had a lot of problems with this and I only solved it by adding `act`, otherwise we will see a lot of warnings on the console like this one.

<img width="600" alt="Screen Shot 2020-09-24 at 18 32 39" src="https://user-images.githubusercontent.com/13750819/94202595-543b2a80-fe94-11ea-9e95-ac46ad1eef0f.png">

I think it makes sense since we will be running all the timeouts that will cause some rendering.